device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/mqtt/MqttIotHubConnection.java

```diff
@@ -492,7 +492,6 @@ public void onMessageArrived(int messageId)
                     transportMessage.setMessageCallbackContext(this.config.getDeviceTelemetryMessageContext(transportMessage.getInputName()));
                     break;
                 case UNKNOWN:
-                case CBS_AUTHENTICATION:
```

Comment:
never happens over mqtt

Reply:
Is this also related to the MessageType change?

https://github.com/Azure/azure-iot-sdk-java/pull/1315/files#diff-86c3b84a005f2abf645a37ba3d3494929e4d00b630bbe03aca65554a6ebf74efL12